scene/3d/xr_nodes.cpp

```diff
@@ -582,6 +588,20 @@ XRPositionalTracker::TrackerHand XRController3D::get_tracker_hand() const {
 	return tracker->get_tracker_hand();
 }
 
+PackedStringArray XRController3D::get_configuration_warnings() const {
+	PackedStringArray warnings = XRNode3D::get_configuration_warnings();
+
+	if (is_visible() && is_inside_tree()) {
```

Comment:
Something we should change here, and probably in other places, is to add a check if there is a parent.

If we're creating a subscene with XRController3D as the root node, we're currently getting our warning, even if we're later on adding this subscene as a child scene to an XROrigin3D node. 
Not really related to this PR but if we're changing this code, this is a long time nitpick I've been wanting to resolve, might as well take it along :)

Reply:
I've modified the logic to warn for XRNode3D and XRCamera3D when it's got a parent which isn't an XROrigin3D. The message is now:
- XRNode3D may not function as expected without an XROrigin3D node as its parent.
- XRCamera3D may not function as expected without an XROrigin3D node as its parent.